core/src/main/java/tc/oc/pgm/controlpoint/ControlPoint.java

```diff
@@ -348,6 +348,11 @@ private void dominateAndFireEvents(@Nullable Competitor dominantTeam, Duration d
     if (oldCapturingTeam != this.capturingTeam) {
       this.match.callEvent(
           new CapturingTeamChangeEvent(this.match, this, oldCapturingTeam, this.capturingTeam));
+      ScoreMatchModule scoreMatchModule = this.getMatch().getModule(ScoreMatchModule.class);
+      // Gives a set number of Owner points to a team when captured.
+      if (this.capturingTeam != null && scoreMatchModule != null) {
+        scoreMatchModule.incrementScore(this.capturingTeam, this.getDefinition().getPointsOwner());
+      }
```

Comment:
The points are added on capture but never removed when uncaptured, see:

https://github.com/OvercastNetwork/ProjectAres/blob/master/PGM/src/main/java/tc/oc/pgm/controlpoint/ControlPoint.java#L349-L353